graphql/schema.graphql

```diff
@@ -105,6 +105,7 @@ type Mutation {
   attachProjectToRepo(projectId: String! @requireProjectAccess(access: EDIT)): Project!
   detachProjectFromRepo(projectId: String! @requireProjectAccess(access: EDIT)): Project!
   forceRepotrackerRun(projectId: String! @requireProjectAccess(access: EDIT)): Boolean!
+  deactivateStepbackTasks(projectId: String! @requireProjectAccess(access: EDIT)): Boolean!
```

Comment:
Are there tests we can write for this?

Reply:
The model function itself is tested so I'm not sure that this requires its own test, since it's just a wrapper